tests/src/python/test_provider_postgres.py

```diff
@@ -1141,6 +1143,32 @@ def testCurveToMultipolygon(self):
         self.assertEqual(g.childCount(), 1)
         self.assertTrue(g.childGeometry(0).vertexCount() > 3)
 
+    def testConcurrency(self):
```

Comment:
What do you think about moving this test to the base provider test suite, so that all providers are checked for this issue?

Reply:
Good idea. Let's see if that works easily.